### DISCUSSION TITLE
full touchscreen support for touch laptops?

### DISCUSSION THREAD

**Discussion by CoolGuy12490** [ppy/osu#14645] [Ideas] [answered]
add touchscreen support to osu!lazer/osu!next like with regular osu!stable

**Comment by peppy** [comment, marked as answer]
What's the actual issue you're facing

**Reply by CoolGuy12490** [reply]
When I use touchscreen on my laptop, the input gets mapped to a super small chunk of the display going to the up left corner
Its basically when I touch the screen on the main menu, it goes to settings
This issue is on osu!lazer

**Reply by peppy** [reply]
Can you try turning off raw input?

**Reply by CoolGuy12490** [reply]
ok let me do that

**Reply by CoolGuy12490** [reply]
oh thanks!
on osu!stable it lets you do touch with raw input

**Reply by peppy** [reply]
For reference, what hardware are you using? PC specs or model so we can figure which touch APIs it is using.
